### PR TITLE
fix get reboot required

### PR DESCRIPTION
**Pull request by pablomendezroyo**
fix get reboot required using a host script instead

### CODE REVIEW

File: packages/admin-ui/src/components/NotificationsMain.tsx [hunk 1]
```diff
@@ -52,9 +52,8 @@ export default function NotificationsView() {
       id: "hostReboot",
       linkText: "Reboot",
       linkPath: systemPathName + "/" + systemSubPaths.power,
-      body:
-        "**Dappnode host reboot required.** Click **Reboot** to reboot the host and apply the changes",
-      active: rebootHostIsRequired
+      body: `**Dappnode host reboot required.** Click **Reboot** to reboot the host and apply the changes. The following packages will be updated: ${rebootHostIsRequired?.pkgs}`,
```
Comment (dappnodedev) [inline packages/admin-ui/src/components/NotificationsMain.tsx:55]: Will the user be redirected to the system page where they can trigger the reboot?

**Review by dappnodedev — APPROVED** [review]

File: packages/hostScripts/hostScripts/reboot_required.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Script to check weather a reboot is required or not.
+
+# Exit on error
+set -e
+
+if [ -f /var/run/reboot-required ]; then
+  # print reboot required and pkgs that require reboot
+  if [ -f /var/run/reboot-required.pkgs ]; then
+    # put pkgs in a string comma separated, if is just 1 pkg, do not add comma
+    pkgs=$(cat /var/run/reboot-required.pkgs | tr '\n' ',' | sed 's/,$//')
```
Comment (Marketen) [inline packages/hostScripts/hostScripts/reboot_required.sh:12]: are we 100% sure that all machines will have `tr` and `sed`?

Comment (pablomendezroyo) [reply packages/hostScripts/hostScripts/reboot_required.sh:12]: we cam assume that

File: packages/admin-ui/src/components/NotificationsMain.tsx (hunk 1 above)
Comment (pablomendezroyo) [reply packages/admin-ui/src/components/NotificationsMain.tsx:55]: yes!

**Pull request merged**